app/controllers/show_projects_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ShowProjectsController < ApplicationController
+  
+  def index
+    @projects = Project.in_current_season.where(aasm_state: :accepted)
```

Comment:
If I recall correctly, AASM automatically adds a scope for each state. So you could shorten this to the even more expressive form of:
```ruby
@projects = Project.in_current_season.accepted
```